src/inmanta/server/services/compilerservice.py

```diff
@@ -376,6 +376,14 @@ async def setup() -> AsyncIterator[Awaitable[Optional[data.Report]]]:
                 compile_data_json_file.name,
             ]
 
+            if self.request.partial:
+                cmd.append("-p")
```

Comment:
```suggestion
                cmd.append("--partial")
```
I think we only support a long option, don't we?

Reply:
Hmm, then I think we diverged from the document, I don't see `-p` in the code except for the profiler, which might actually be the reason we couldn't use it for this.

Reply:
Though I think the profiler is an option of the `inmanta` command, not `inmanta compile`.

Reply:
The `-p` option was already taken to enable profiling. I updated the tickets, but I didn't update the design document.